compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/AnnotationDesugar.java

```diff
@@ -107,7 +220,7 @@
  *
  * @since 0.965.0
  */
-public class AnnotationDesugar {
+public class AnnotationDesugar extends BLangNodeVisitor {
```

Comment:
Isn't `BLangNodeVisitor` deprecated? Why aren't we going with `BLangNodeAnalyzer`?

Reply:
Removed this phase and move this logic to the closure generator phase.